src/core/public/chrome/ui/header/collapsible_nav.tsx

```diff
@@ -154,17 +154,18 @@ export function CollapsibleNav({
     workspace: WorkspaceAttribute | null | undefined,
     categorizedLinks: Record<string, ChromeNavLink[]>
   ) {
-    // plugins are in this dictionary
-    const pluginsDictionary = categorizedLinks.opensearch;
-    if (!pluginsDictionary) return categorizedLinks;
-
     const features = workspace?.features ?? [];
-    const newPluginsDictionary = pluginsDictionary.filter((item) => features.indexOf(item.id) > -1);
-    if (newPluginsDictionary.length === 0) {
-      delete categorizedLinks.opensearch;
-    } else {
-      categorizedLinks.opensearch = newPluginsDictionary;
-    }
+
+    Object.keys(categorizedLinks).forEach((category) => {
```

Comment:
I remember we agreed in last meeting that we don't do filter if current workspace is not set.